events.html

```diff
@@ -137,23 +137,64 @@ <h1 class="title">
                     <div class="field">
                         <div class="control has-icons-left has-icons-right">
                             <input id="game-desc" class="input is-medium" type="text" placeholder="Small">
-                            
                             <span class="icon is-right">
                                 <i class="fas fa-check fa-sm"></i>
                             </span>
                         </div>
                     </div>
-
+                   <!-- date select -->
                     <div class="field">
                         <div class="control has-icons-left has-icons-right">
                             <input id="game-date" class="input is-medium" type="date" placeholder="Normal">
-                            
                             <span class="icon is-medium is-right">
                                 <i class="fas fa-check"></i>
                             </span>
                         </div>
+                        </div>
+
+                        <!-- Hours -->
+                        <div class="field">
+                            <div class="control has-icons-left is-pulled-left">
+                              <div class="select is-success">
+                                <select>
+                                  <option selected>Hour</option>
+                                  <option>1</option>
```

Comment:
Could these be updated to have a 0 in front of the single digit numbers? It will help with the QR code generator